cli/src/torii/routing.rs

```diff
@@ -32,9 +36,11 @@ pub(crate) async fn handle_queries<W: WorldTrait>(
     query_validator: Arc<IsQueryAllowedBoxed<W>>,
     pagination: Pagination,
     request: VerifiedQueryRequest,
-) -> Result<Scale<VersionedQueryResult>, query::Error> {
-    let valid_request = request.validate(&wsv, &query_validator)?;
-    let result = valid_request.execute(&wsv)?;
+) -> Result<Scale<VersionedQueryResult>, warp::http::Response<warp::hyper::Body>> {
+    let valid_request = request
+        .validate(&wsv, &query_validator)
+        .map_err(into_reply)?;
+    let result = valid_request.execute(&wsv).map_err(into_reply)?;
```

Comment:
No need to `map_err` as `query::Error` will be recovered into `Response` in `handle_rejection`.

However, if we have every handler have `Response<Body>` Err variant,
`handle_rejection` will be one-line -- just pick `Response<Body>`.
We could go this way

Reply:
You mean impl From? I would do that, but it introduced a bidirectional dependency. Since this was the only place where `impl Reply` was used, I decided to inline the implementation.

Reply:
Response<Body> actually looks interesting.